upgrade/scripts/upgrade/csm-upgrade.sh

```diff
@@ -156,6 +156,12 @@ fi
 # Restart CFS deployments to avoid CASMINST-6852
 "${basedir}/../common/restart-cfs.sh"
 
+# Update test/CLI RPMs on NCNs
+"${basedir}/util/upgrade-test-rpms.sh"
+
+# Wait 10 seconds to make sure all of the restarted Goss servers are started up on the NCNs
+sleep 10
```

Comment:
This is pretty arbitrary, can we just check if the service has started instead or is that too difficult for this context?

Reply:
Since it would be checking it on all of the remote servers, it seemed simpler to do it this way. But I guess I could just modify the upgrade-test-rpms script to do that.